docker-compose.yml

```diff
@@ -17,6 +17,34 @@ services:
             APP_SECRET:
     pattern-library:
         image: ${DOCKER_NAMESPACE}/pattern-library:${REVISION_PATTERN_LIBRARY}
+    content-store_fpm:
```

Comment:
We'll need multiple content stores, so this should be `scholarly-articles_fpm` etc.